### PR TITLE
Additional OVF import e2e tests

### PR DESCRIPTION
**Pull request by zoran15**

### CODE REVIEW

File: gce_ovf_import_tests/test_suites/ovf_import_test_suite.go [hunk 1]
```diff
@@ -71,10 +78,27 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
...
+	if err != nil {
+		os.Exit(1)
+	}
+	startupScriptWindowsSingleDisk, err := ioutil.ReadFile(
+		"daisy_integration_tests/scripts/post_translate_test.ps1")
+	if err != nil {
+		os.Exit(1)
+	}
+	startupScriptWindowsTwoDisks, err := ioutil.ReadFile(
+		"gce_ovf_import_tests/scripts/ovf_import_test_windows_2k12_r2_two_disks.ps1")
```
Comment (dntczdx) [inline gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: May I have your help to understand why ovf_import_test_windows_2k12_r2_two_disks.ps1 is under different folder from  post_translate_test.ps1? And why it's specifically for 2k12? (I read the file but don't see an obvious signal why it's only for 2k12)

File: gce_ovf_import_tests/test_suites/ovf_import_test_suite.go [hunk 2]
```diff
@@ -85,11 +109,67 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 				Labels:        "lk1=lv1,lk2=kv2",
 				Project:       testProjectConfig.TestProjectID,
 				Zone:          testProjectConfig.TestZone,
-				MachineType:   "n1-standard-2",
+				MachineType:   "n1-standard-1",
```
[the remark this answers is not in the document]
Comment (zoran15) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: To test different machine types in different tests and make sure there's no overlap between tests as much as possible.

File: gce_ovf_import_tests/test_suites/ovf_import_test_suite.go (hunk 1 above)
Comment (zoran15) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: 2k12_r2 OVA has two disks and needs additional code to check that. 2k16 has just one disk so we can use standard post translate test to validate it.

File: cli_tools/gce_ovf_import/README.md [hunk 3]
```diff
@@ -94,6 +94,7 @@ go get github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_impo
 + `-disable-cloud-logging` do not stream logs to Cloud Logging
 + `-disable-stdout-logging` do not display individual workflow logs on stdout
 + `-no-guest-environment` Google Guest Environment will not be installed on the image
++ `-node-affinity-label` Node affinity label used to determine sole tenant node to schedule this instance on. Label is of the format: <key>,<operator>,<value>,<value2>... where <operator> can be one of: IN, NOT. For example: workload,IN,prod,test is a label with key 'workload' and values 'prod' and 'test'. This flag can be specified multiple times for multiple labels.
```
[the remark this answers is not in the document]
Comment (zoran15) [reply cli_tools/gce_ovf_import/README.md]: missed README for sole tenant node feature that I sneaked into this PR

File: gce_ovf_import_tests/test_suites/ovf_import_test_suite.go (hunk 1 above)
Comment (dntczdx) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: I understand this one has two disks and need additional code for that. My question is whether it also works for 2016 and other windows versions. We may have the requirement to produce more e2e tests, and may reuse this script for different versions, if it works.

Comment (zoran15) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: It should work, though I don't see us adding more windows 2012/2016 tests with two disks specifically.

Comment (dntczdx) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: Then I would suggest to change the file name to "ovf_import_test_windows_two_disks" to make it accurate. It's your call

Comment (zoran15) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: Done

File: gce_ovf_import_tests/test_suites/ovf_import_test_suite.go [hunk 4]
```diff
@@ -71,10 +78,27 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	suffix := utils.RandString(5)
 	logger.Printf("Running TestSuite %q", testSuite.Name)
 
-	startupScript, err := ioutil.ReadFile("gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	startupScriptUbuntu3disks, err := ioutil.ReadFile(
+		"gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	if err != nil {
+		os.Exit(1)
```
Comment (adjackura) [inline gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: Just an exit is a bit abrupt and hard to diagnose, at least log the error

Comment (zoran15) [reply gce_ovf_import_tests/test_suites/ovf_import_test_suite.go]: Added logging and refactored script loading to a separate func

**Pull request merged**